frontend/public/components/graphs/gauge.tsx

```diff
@@ -31,11 +31,17 @@ export const GaugeChart: React.FC<GaugeChartProps> = ({
   label = data ? humanize(data.y).string : 'No Data',
   secondaryTitle = usedLabel,
   className,
+  totalUsedValue,
 }) => {
   const [ref, width] = useRefWidth();
   const ready = !error && !loading;
   const status = loading ? 'Loading' : error;
-  const labels = ({ datum: { x, y } }) => (x ? `${x} ${usedLabel}` : `${y} ${remainderLabel}`);
+  const labels = ({ datum: { x, y } }) => {
+    const totalUsedLabel = totalUsedValue
+      ? `${totalUsedValue} ${usedLabel}`
+      : `${humanize(y).string} ${usedLabel}`;
+    return x ? `${x} ${remainderLabel}` : totalUsedLabel;
```

Comment:
If we go this way, I'd expect to the used and remaining tooltip to show the same kind of data. So it would be actual used and actual remaining. Actual used and percentage remaining is odd.